profiles/esp-idf.code-profile

```diff
@@ -0,0 +1 @@
false}]\",\"extensions.trustedPublishers\":\"{\\\"espressif\\\":{\\\"publisher\\\":\\\"espressif\\\",\\\"publisherDisplayName\\\":\\\"Espressif Systems\\\"}}\",\"workbench.view.extension.espIdfHints.state.hidden\":\"[{\\\"id\\\":\\\"errorHints\\\",\\\"isHidden\\\":false}]\",\"workbench.view.extension.idfViewController.state.hidden\":\"[{\\\"id\\\":\\\"idfCommands\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"idfSearchResults\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"idfPartitionExplorer\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"idfAppTracer\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"idfAppTraceArchive\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"espRainmaker\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"espEFuseExplorer\\\",\\\"isHidden\\\":false}]\",\"workbench.welcomePage.walkthroughMetadata\":\"[[\\\"espressif.esp-idf-extension#espIdf.walkthrough.basic-usage\\\",{\\\"firstSeen\\\":1743665334705,\\\"stepIDs\\\":[\\\"introduction\\\",\\\"extension-configuration\\\",\\\"create-example-project\\\",\\\"setup-project\\\",\\\"build-flash-monitor\\\",\\\"additional-resources-and-ui\\\"],\\\"manaullyOpened\\\":false}],[\\\"espressif.esp-idf-extension#espIdf.walkthrough.advanced\\\",{\\\"firstSeen\\\":1743665334705,\\\"stepIDs\\\":[\\\"advanced-introduction\\\",\\\"troubleshooting\\\",\\\"sdk-configuration\\\",\\\"debugging\\\",\\\"component-installation\\\",\\\"app-size-analysis\\\",\\\"partition-table\\\",\\\"unit-testing\\\",\\\"multi-project\\\",\\\"conclusion\\\"],\\\"manaullyOpened\\\":false}],[\\\"ms-vscode.cpptools#cppWelcome\\\",{\\\"firstSeen\\\":1743665943988,\\\"stepIDs\\\":[\\\"awaiting.activation.mac\\\",\\\"awaiting.activation.linux\\\",\\\"awaiting.activation.windows\\\",\\\"awaiting.activation.windows10\\\",\\\"awaiting.activation.windows11\\\",\\\"no.compilers.found.mac\\\",\\\"no.compilers.found.linux\\\",\\\"no.compilers.found.windows\\\",\\\"no.compilers.found.windows10\\\",\\\"no.compilers.found.windows11\\\",\\\"verify.compiler.mac\\\",\\\"verify.compiler.linux\\\",\\\"verify.compiler.windows\\\",\\\"verify.compiler.windows10\\\",\\\"verify.compiler.windows11\\\",\\\"create.cpp.file\\\",\\\"relaunch.developer.command.prompt.windows\\\",\\\"run.project.mac\\\",\\\"run.project.linux\\\",\\\"run.project.windows\\\",\\\"customize.debugging.linux\\\",\\\"customize.debugging.windows\\\",\\\"customize.debugging.mac\\\"],\\\"manaullyOpened\\\":false}]]\",\"workbench.view.extension.test.state.hidden\":\"[{\\\"id\\\":\\\"workbench.view.testing\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"workbench.view.testCoverage\\\",\\\"isHidden\\\":false}]\",\"workbench.panel.alignment\":\"center\",\"fileBasedRecommendations/promptedRecommendations\":\"{\\\"c\\\":[\\\"ms-vscode.cpptools-extension-pack\\\"]}\",\"workbench.view.extension.cmake-view.state.hidden\":\"[{\\\"id\\\":\\\"cmake.projectStatus\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"cmake.outline\\\",\\\"isHidden\\\":false},{\\\"id\\\":\\\"cmake.pinnedCommands\\\",\\\"isHidden\\\":false}]\"}}"}
```

Comment:
Some settings are specific to your local setup `C:\\\\\\\\Users\\\\\\\\radur\\\\\\\\esp\\\\\\\\v5.4.1\\\\\\\\esp-idf\\\` and I would consider many of the extension included are not necessary for ESP-IDF development.

Why does the template profile include Themes? Why CMake extension?

We should have another profile template for Clang extension too.